source/TextBlade.ConsoleRunner/Game.cs

```diff
@@ -67,6 +67,8 @@ public void Run()
     {
         try {
             LoadGameOrStartNewGame();
+            if (_currentLocation is null)
+                throw new InvalidOperationException("Game has not been set up");
```

Comment:
Maybe "game didn't start with a location" or some other message makes sense here.

Also, please make sure you add curly braces and follow the existing code style.